settings/common.py

```diff
@@ -23,6 +23,7 @@
 SITE_VARIABLES = {
```

Comment:
I would prefer to use environment variables for everything as far-as-possible (12-factor). But overriding entire `SITE_VARIABLES` works as-well.